test-network-k8s/scripts/pkgk8scc.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh -l
```

Comment:
I was expecting the `package_k8s_chaincode` function to use this script but it doesn't seem to. Why is there a copy here instead of using [the original](https://github.com/hyperledgendary/package-k8s-chaincode-action/blob/main/pkgk8scc.sh)? Hopefully the `peer` command will be able to handle more packages soon :)

Reply:
This is lame ... but: 

- pkgk8scc.sh will write outputs into $PWD.  It needs a flag and/or arg to set the -o "output" path for the tgz

- Before adding this to scripts/, I was burned a bit by having routines that need to download routines and then run them locally (albeit in a container...)   Little questions like ... "what happens if the remote script is broken?  Someone will call me and complain that test network no longer works..."  and "how can we end up with a one-liner installation in a docs / guide, that does not ask users to download a script before running?" ... etc. 

- I'm not sold on the script being in this code line at all.  It is *very* convenient, but better homed in the builder docs / guide, or ultimately landing as a core routine in the [fabric-cli]() (vaporware) or [peer](https://github.com/hyperledger/fabric/issues/3368) binary.

Reply:
One day `fabric-cli` will condense!